_gsocblogs/2022/blog_CompilerResearch-ManishKausik.md

```diff
@@ -0,0 +1,28 @@
+---
+project: Compiler-Research
+title: Add Initial Integration of Clad with Enzyme 
+author: Manish Kausik H
+photo: blog_authors/ManishKausik.jpg
+date: 24.07.2022
+year: 2022
+layout: blog_post
+logo: compiler-research-logo.png   
+intro: |
+  A blog that summarises my experience with GSoC so far and also gives a brief of the work I've done. 
+---
+## What Interested me to apply for this Project?
+Ever since as I was introduced to Compiler Design as a part of my university course work, I have been fascinated by this subject and was always eager to explore more about it. Naturally
+I came across the Clang-LLVM Infrastructure and was interested in learning more about it. I found the Compiler-Research group while looking out for some cool projects related to the Clang-LLVM Infrastructure, and as the GSoC Organizations were announced, I got in touch with the Group members hoping to contribute in their work. This project was suggested to me by my mentor Vassil, as its a nice fit for a beginner to get learning about the Clang-LLVM Compiler Infrastructure.
+
+Here is my [Project Proposal](https://compiler-research.org/assets/docs/Manish_Kausik_H_Proposal_2022.pdf).
+
+## GSoC Community Bonding Period
+As the Community Bonding period started, I got into fixing a few bugs in the Clad repository, just to get to know the codebase better. I was successful in fixing some, but was not successful in fixing a few others. Meanwhile I also presented my Project Proposal to the rest of the Compiler Research community([Slides](https://compiler-research.org/assets/presentations/CaaS_Weekly_01_05_2022_Manish_Add_Initial_Integration_of_Clad_with_Enzyme.pdf)) and got their feedback on how I intend to take the project forward. I started participating in the weekly meetings of the Community and learnt(and still learning) about the various projects that the community members are doing.
```

Comment:
We should also have a paragraph introducing automatic differentiation and compiler-based automatic differentiation.